### DISCUSSION TITLE
[2536840]GINKA{本体}

### DISCUSSION THREAD

**Discussion by ZiBiKai04** [pjy612/SteamManifestCache#513] [Share]
`appid:2536840`
keys:
```
2536841;9b3b47192b004e86cd514d61f138b8c45ce8f6cd0008406d4011dbb6f4a84e59
2536842;6f83c22b4b5e9d0c07fbc00625fc5285bd9046017f2dbdffce4e6e6604269328
```
sai找不到了 没办法查看code
邮箱: jsps1064502@gmail.com
[GINKA-2536840.zip](https://github.com/pjy612/SteamManifestCache/files/14373438/GINKA-2536840.zip)

**Comment by pjy612** [comment]
* 密钥错误 #26
* https://steamdb.info/app/2536840/depots/

**Reply by ZiBiKai04** [reply]
请问一下 是steamdb上的三个depot id密钥都要吗？第一次用不太懂

**Reply by ZiBiKai04** [reply]
我是缺了剩下的两个depot id没给密钥吗？

**Reply by pjy612** [reply]
Shared Install 的不用发
其他的 需要发
至少 密钥要和 清单对应

**Reply by ZiBiKai04** [reply]
已修改

**Reply by ZiBiKai04** [reply]
@pjy612 大佬那请问您方便发内测给我吗

**Reply by pjy612** [reply]
号太新

**Reply by ZiBiKai04** [reply]
我之前那只GitHub不知道咋了 发了讨论是locked #506、#507都是我发的 不过被锁住不公开

**Reply by pjy612** [reply]
大概率格式不对 又不改被删了
